gradle-modules/src/main/kotlin/org/sonarsource/cloudnative/gradle/CommonSettingsPlugin.kt

```diff
@@ -0,0 +1,145 @@
+/*
+ * SonarSource Cloud Native Gradle Modules
+ * Copyright (C) 2024-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the Sonar Source-Available License Version 1, as published by SonarSource SA.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the Sonar Source-Available License for more details.
+ *
+ * You should have received a copy of the Sonar Source-Available License
+ * along with this program; if not, see https://sonarsource.com/license/ssal/
+ */
+package org.sonarsource.cloudnative.gradle
+
+import com.gradle.develocity.agent.gradle.DevelocityConfiguration
+import javax.inject.Inject
+import org.gradle.api.Plugin
+import org.gradle.api.artifacts.dsl.RepositoryHandler
+import org.gradle.api.artifacts.repositories.MavenArtifactRepository
+import org.gradle.api.initialization.Settings
+import org.gradle.api.internal.file.FileOperations
+import org.gradle.api.provider.ProviderFactory
+import org.gradle.internal.extensions.core.extra
+import org.gradle.kotlin.dsl.assign
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.getByType
+import org.gradle.kotlin.dsl.repositories
+
+open class CommonSettingsPlugin
+    @Inject
+    constructor(
+        private val fileOperations: FileOperations,
+    ) : Plugin<Settings> {
+        override fun apply(settings: Settings) {
+            settings.configureRepositories()
+
+            settings.applyPlugins()
+
+            settings.gradle.allprojects {
+                // this value is present on CI
+                val buildNumber: String? = System.getProperty("buildNumber")
+                // TODO: is it still required to put this value in extra?
+                project.extra["buildNumber"] = buildNumber
```

Comment:
I guess is not needed anymore, please verify

Reply:
Yes, removed